Files/MultilingualResources/Files.hu-HU.xlf

```diff
@@ -3176,55 +3176,55 @@
         </trans-unit>
         <trans-unit id="DecompressArchiveDialog.Title" translate="yes" xml:space="preserve">
           <source>Select a destination and extract files</source>
-          <target state="new">Select a destination and extract files</target>
+          <target state="translated">Válasszon egy célhelyet, és csomagolja ki a fájlokat</target>
         </trans-unit>
         <trans-unit id="DecompressArchiveDialogBrowseFolder.Content" translate="yes" xml:space="preserve">
           <source>Browse</source>
-          <target state="new">Browse</target>
+          <target state="translated">Keresés</target>
```

Comment:
```suggestion
          <target state="translated">Kiválasztás</target>
```
Other alternatives: `Választás`, `Mappaválasztás`

This is a text of the button for opening folder picker dialog:

![image](https://user-images.githubusercontent.com/42123156/127279536-d6d47e68-cf6e-442c-b4e9-70fdc96a1c91.png)